Add csp report URI for securedrop.org

emkll opened the pull request:
Adds the CSP report uri for securedrop.org

Now that the freedom.press uri works, we can test it with securedrop.org in our next staging deployment. Report URI also allows us to filter by domain name.

emkll committed:
Add csp report URI for securedrop.org

conorsch reviewed (changes requested):
Two changes:

* more explicit string setting
* support env var

will edit and push up

Commit:
Supports env var for CSP report URI

Also added a comment explicitly stating the value must be a string,
since that's what the docs [0] indicate.

[0] https://django-csp.readthedocs.io/en/latest/configuration.html
